### PR TITLE
Adds Tetun Translations and IGNORE_BROWSER_LANGUAGE env Setting

### PR DESCRIPTION
**Pull request by sandroroux**

### CODE REVIEW

File: canoe-project-default.js [hunk 1]
```diff
@@ -23,5 +23,4 @@ module.exports = {
     REQUIRE_LOGIN: false,
     PROFILE_LINK: "profile",
     LANGUAGES: ["en", "fr", "tet"],
```
Comment (PeteCoward) [inline canoe-project-default.js:25]: surely you should add `IGNORE_BROWSER_LANGUAGE: false` here?

Comment (PeteCoward) [reply canoe-project-default.js:25]: and adding `LANGUAGES: ["en"]` which I should have done would avoid the need for the check in Translation.js maybe

Comment (sandroroux) [reply canoe-project-default.js:25]: I have `IGNORE_BROWSER_LANGUAGE` in canoe-environment. How should I decide if a setting belongs in -project or -environment?

**Review by PeteCoward — APPROVED** [review]
oh, they were there already. Then the code to guard against it in Translations is mostly unnecessary, approved anyway

**Pull request merged**